src/components/product/ProductCard/product-card.scss

```diff
@@ -1,78 +1,101 @@
 @import "src/styles/scaffold";
 
-.product-card[data-store-card] {
-  position: relative;
-  display: flex;
-  flex-direction: column;
-  min-width: 14rem;
-  height: 100%;
-  padding: var(--fs-spacing-1) var(--fs-spacing-1) var(--fs-spacing-2);
-  border: var(--fs-border-width-default) solid transparent;
-  border-radius: var(--fs-border-radius-default);
-  transition: box-shadow .5s ease, border .5s ease;
-
-  &:hover,
-  &:focus {
-    background-color: var(--fs-color-neutral-0);
-  }
+/* stylelint-disable */
```

Comment:
What's up with disabling the Stylelint here? Feels like it's going to be easy not to follow its rules later in future changes to this file.

Reply:
Better see [this other comment](https://github.com/vtex-sites/base.store/pull/425#discussion_r836595761)!